units/XNS0302/XNS0302_unit.bp

```diff
@@ -235,7 +235,7 @@ UnitBlueprint {
         TechLevel = 'RULEUTL_Secret',
         ToggleCaps = {
         },
-        UnitName = '<LOC XNS0302_name>Juggernaut',
+        UnitName = '<LOC XNS0302_name>Juggernaut class',
```

Comment:
Shouldn't it be "Class" with capital C like the others?

Reply:
Good catch